c7n/filters/core.py

```diff
@@ -330,6 +331,19 @@ def process_value_type(self, sentinel, value):
             # comparisons is intuitively wrong.
             return value, sentinel
 
+        elif self.vtype == 'cidr_size':
+            v = []
+            for r in value:
+                try:
+                    cidr_size = ipaddress.ip_network(unicode(r.get('CidrIp'))).prefixlen
+                except ipaddress.AddressValueError:
+                    cidr_size = 0
+                if v > 0:
+                    v.append(cidr_size)
+            print('Sentinel: {}\tConverted Value: {}'.format(sentinel, v))
```

Comment:
log and run verbose, if testing we have a capture_log method for testing output, or debugging.